docker.py

```diff
@@ -180,25 +190,31 @@ def get_numpy_choices():
 RUN cd "$PYENV_ROOT" && git pull && cd - && env CFLAGS="-fPIC" PYTHON_CONFIGURE_OPTS="--enable-shared" pyenv install {python_ver}
 RUN pyenv global {python_ver}
 RUN pyenv rehash
+
+ENV CUTENSOR_URL=https://developer.download.nvidia.com/compute/cuda/repos/ubuntu{cutensor_os_ver}/x86_64
```

Comment:
This is only applicable to -pyenv configurations.

Reply:
Fixed!